packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -113,11 +113,7 @@ function UncontrolledInnerBlocks( props ) {
 		return blockList;
 	}
 
-	return (
-		<div className="block-editor-inner-blocks" ref={ forwardedRef }>
```

Comment:
`forwardedRef` sounds like a leftover from a class component. Ideally, we should be using the `forwardRef` HoC to get a ref.